pkg/microservice/aslan/core/system/handler/router.go

```diff
@@ -87,6 +87,7 @@ func (*Router) Inject(router *gin.RouterGroup) {
 	{
 		cleanCache.POST("/oneClick", CleanImageCache)
 		cleanCache.GET("/state", CleanCacheState)
+		cleanCache.POST("/setCron", SetCron)
```

Comment:
It may be better to use `/cron` path with `POST` and don't use uppercase letter in url path.